unit_tests/test_lib_charms_ovn_charm.py

```diff
@@ -423,10 +445,10 @@ def test_configure_ovs(self):
         self.get_ovs_hostname.return_value = 'fake-ovs-hostname'
         self.patch_target('check_if_paused')
         self.check_if_paused.return_value = ('some', 'reason')
-        self.target.configure_ovs('fake-sb-conn-str')
+        self.target.configure_ovs('fake-sb-conn-str', True)
```

Comment:
I understand what the second argument is (whether the mlockall option has changed or not) but I'm a bit lost here why it would be True the first time, False the second time, then True again. Can you please add inline comments to elaborate how you made that choice for the sake of testing? Thanks!

Reply:
Well the reasoning isn't very exciting. The only logic behind the choice was that it needed a parameter added so I figured I'd randomize it a bit.

Reply:
I think it would probably make sense to add a check that service_restart is called or not based on the parameter.

Reply:
I think this is resolved now with the new checks I've added. Let me know if you'd still like the comments added but I think maybe the added checks explain it without needing comments now.